applications/Rekall/views/flows/collection_view.html

```diff
@@ -2,6 +2,16 @@
 
 <h3 id="title"></h3>
 
+<nav aria-label="breadcrumb" role="navigation">
+  <ol class="breadcrumb">
+      <li class="breadcrumb-item"><a href="{{=URL(c='default', f='index')}}">Home</a></li>
+    <li class="breadcrumb-item"><a href="{{=URL(c='flows', f='inspect_list')}}?client_id={{=client_id}}">{{=client_id}}</a></li>
+      <li class="breadcrumb-item"><a href="{{=URL(c='flows', f='description')}}?client_id={{=client_id}}&flow_id={{=flow_id}}">{{=flow_id}}</a></li>
```

Comment:
You can add these parameters to the var keyword arg of the URL() function:
http://www.web2pyref.com/reference/url-web2py-function